.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "chartboost-mediation-android-sdk-demo"]
```

Comment:
@cb-jpadilla we probably didn't want the submodule copied over.

Reply:
Oh, I see, will adjust the script. 📓